components/view_image/popover_bar/__snapshots__/popover_bar.test.jsx.snap

```diff
@@ -23,7 +23,6 @@ exports[`components/view_image/popover_bar/PopoverBar should match snapshot for
   >
     <a
       className="text"
-      download="img.png"
```

Comment:
Do we need a test that covers the new code path?

Reply:
It's already covered by the existing tests for internal links. Previously, we had this set for both internal and external links, but now it's only set for internal ones.